pkg/api/vm/handler.go

```diff
@@ -717,6 +719,37 @@ func (h *vmActionHandler) createVMImages(templateVersion *harvesterv1.VirtualMac
 	return nil
 }
 
+// Since the LH volume creation and replica scheduling are asynchronous
+// Even the CreateVolume csi call success, the replica scheduling may still fail
+// However for the sc with VolumeBindingMode is Immediate
+// we could check annotation for corresponding error if it exists
+func (h *vmActionHandler) checkForConventionSC(pvc *corev1.PersistentVolumeClaim) error {
+	sc, err := h.storageClassCache.Get(*pvc.Spec.StorageClassName)
```

Comment:
the `pvc.Spec.StorageClassName` is allowed to be nil, better to check != nil first

Reply:
Thanks, @w13915984028. I added checking based on your suggestion.